engine/image-loader/t/image-loader-test.lisp

```diff
@@ -5,26 +5,19 @@
 (deftest make-image-test
   (ok (instance-of-p
           'lbge.image-loader::image
-         (make-image :width 50 :height 50 :channels "rgb8" :data "raw-data"))))
+         (make-image :width 50 :height 50 :channels "rgba8" :data (vector 1 2 3 4)))))
```

Comment:
Another way to create a vector would be `#(1 2 3 4)`. I think it just exapands to that exact function.